Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/sync.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+  namespace: "gatekeeper-system"
+spec:
+  sync:
+    syncOnly:
+      - group: ""
+        version: "v1"
+        kind: "Namespace"
+      - group: ""
+        version: "v1"
+        kind: "Pod"
+      - group: ""
+        version: "v1"
+        kind: "Service"
+      - group: "extensions"
+        version: "v1beta1"
```

Comment:
Can you just have ingress in this file as other types are not required for sync? I think ingress is the only referential policy we have. Please check.